scene/2d/animated_sprite.h

```diff
@@ -98,6 +98,7 @@ class SpriteFrames : public Resource {
 		ERR_FAIL_COND_V(p_idx < 0, Ref<Texture>());
 
 		const Map<StringName, Anim>::Element *EN = animations.find(E->get().normal_name);
+		ERR_FAIL_COND_V(!EN, Ref<Texture>());
```

Comment:
Not sure if this should be an error condition or just silently return an empty reference. I guess it's an error since `!E` is a fail condition...